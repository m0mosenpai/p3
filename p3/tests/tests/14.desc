Multiple cd usage with local variables