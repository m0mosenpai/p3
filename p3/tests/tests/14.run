../solution/wsh <tests/14.wsh
