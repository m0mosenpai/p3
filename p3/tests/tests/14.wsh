local a=home
cd /
cd $a
pwd
cd /
local b=usr
cd $b
pwd
cd /
cd $a
pwd
cd /
cd $b
pwd
cd /
cd $a
pwd
cd /
cd $b
pwd
cd /
cd $a
pwd
cd /
cd $b
pwd
cd /
cd $a
pwd
cd /
cd $b
pwd
cd /
cd $a
pwd
cd /
cd $b
pwd
exit
