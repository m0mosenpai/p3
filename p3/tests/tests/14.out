wsh> wsh> wsh> wsh> /home
wsh> wsh> wsh> wsh> /usr
wsh> wsh> wsh> /home
wsh> wsh> wsh> /usr
wsh> wsh> wsh> /home
wsh> wsh> wsh> /usr
wsh> wsh> wsh> /home
wsh> wsh> wsh> /usr
wsh> wsh> wsh> /home
wsh> wsh> wsh> /usr
wsh> wsh> wsh> /home
wsh> wsh> wsh> /usr
wsh> 